src/commands/passthroughMcpServer.ts

```diff
@@ -0,0 +1,274 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import {
+  IActionContext,
+  isUserCancelledError,
+} from "@microsoft/vscode-azext-utils";
+import * as vscode from "vscode";
+import * as crypto from "crypto";
+import { BackendContract } from "@azure/arm-apimanagement";
+import { McpServersTreeItem } from "../explorer/McpServersTreeItem";
+import { ApimService } from "../azure/apim/ApimService";
+import { localize } from "../localize";
+
+interface McpServerConfig {
+  mcpServerName: string;
+  displayName: string;
+  mcpServerUrl: string;
+  apiUrlSuffix: string;
+  protocolType: string;
+  sseEndpoint?: string;
+  messagesEndpoint?: string;
+}
+
+export async function passthroughMcpServer(
+  context: IActionContext,
+  node?: McpServersTreeItem
+): Promise<void> {
+  try {
+    if (!node) {
+      return;
+    }
+
+    const mcpServerName = (
+      await context.ui.showInputBox({
+        prompt: localize(
+          "enterMcpServerName",
+          "Enter the name for the MCP server"
+        ),
+        validateInput: (value: string): string | undefined => {
+          if (!value || value.trim().length === 0) {
+            return localize("nameRequired", "Name is required");
+          }
+          return undefined;
+        },
+      })
+    ).trim();
+
```

Comment:
if user cancel this action, no mcpServerName inputed, will return directly?

Reply:
If user cancelled the input, an UserCancelledError will be thrown. The code will catch this error and do nothing.